apps/website/src/components/layout/footer/Socials.tsx

```diff
@@ -92,6 +97,31 @@ const Socials: React.FC = () => {
               </li>
             ))}
           </ul>
+
+          <p className="mb-2 mt-4">
+            and keep up-to-date with our announcement channels:
+          </p>
+
+          <ul className="flex flex-wrap gap-4">
+            <li>
+              <NotificationsButton
+                className={buttonClasses}
+                openDirectionX="right"
+                openDirectionY="top"
+              />
+            </li>
+            <li>
+              <a
+                className={buttonClasses}
+                href="https://discord.com/channels/548410541991919617/1052380120981180426"
```

Comment:
We should put this link in a config file (also, it needs to be the invite link -- channel links only work for folks already in there)